src/main/java/eva/tasks/Event.java

```diff
@@ -0,0 +1,93 @@
+package eva.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import eva.exceptions.TaskException;
+
+/**
+ * Represents an Event object. An <code>Event</code> has a name, a start time and an end time.
+ * This class extends from the <code>Task</code> class.
+ */
+public class Event extends Task {
+    private LocalDate startTime;
+    private LocalDate endTime;
+
+    /**
+     * Creates an Event object with the specified name, start time and end time.
+     *
+     * @param name name of the event.
+     * @param startTime start time of the event.
+     * @param endTime end time of the event.
+     */
+    private Event(String name, LocalDate startTime, LocalDate endTime) {
+       super(name);
+       this.startTime = startTime;
+       this.endTime = endTime;
```

Comment:
I think the indentation should be 8 here?
```suggestion
        super(name);
        this.startTime = startTime;
        this.endTime = endTime;
```

Reply:
I believe the indentation should be 8 here, you may have missed an extra whitespace while indenting your code.